fe1-web/src/core/objects/Hash.ts

```diff
@@ -1,8 +1,10 @@
 import { sha256 } from 'js-sha256';
 
-import { Base64UrlData } from './Base64Url';
+import { Base64UrlData } from './Base64UrlData';
 import { PublicKey } from './PublicKey';
 
+export type HashState = string;
+
```

Comment:
comment for lower in this file: 
if we change the implementation of `fromStringArray` to take a `String[]` input instead of a `string[]` input, we could now do a `.valueOf()` inside that function and avoid doing it everywhere else in the application :)

Reply:
Oh wow this is great, love it! 🥳😁